

def zomato_bot():
    try:
        pass
    except Exception as e:
        print(f"Error occured in zomato bot: {e}")
        raise e
